lib/Feature/FeatureModelParser.cpp

```diff
@@ -187,16 +187,16 @@ std::unique_ptr<FeatureModel> FeatureModelXmlParser::buildFeatureModel() {
 
 std::unique_ptr<xmlDtd, void (*)(xmlDtdPtr)>
 FeatureModelXmlParser::createDtd() {
-  std::unique_ptr<xmlDtd, void (*)(xmlDtdPtr)> Dtd(
-      xmlIOParseDTD(nullptr,
-                    xmlParserInputBufferCreateMem(DtdRaw.c_str(),
-                                                  DtdRaw.length(),
-                                                  XML_CHAR_ENCODING_UTF8),
-                    XML_CHAR_ENCODING_UTF8),
-      xmlFreeDtd);
-  xmlCleanupParser();
-  assert(Dtd && "Failed to parse DTD.");
-  return Dtd;
+    std::unique_ptr<xmlDtd, void (*)(xmlDtdPtr)> Dtd(
+            xmlIOParseDTD(nullptr,
+                          xmlParserInputBufferCreateMem(DtdRaw.c_str(),
+                                                        DtdRaw.length(),
+                                                        XML_CHAR_ENCODING_UTF8),
+                          XML_CHAR_ENCODING_UTF8),
+            xmlFreeDtd);
+    xmlCleanupParser();
+    assert(Dtd && "Failed to parse DTD.");
+    return Dtd;
```

Comment:
These lines are just reformatted with white space. Was this wrong in the original version? Normally, travis should cache something like this. Or, is your formatter wrongly configured and want's to add spaces here?
By default clang-format should discover the projects formatting config.

Reply:
Fault of my formatter, will revert it.